### PR TITLE
improve performance

### PR DESCRIPTION
**Pull request by kevindiu**
It reduce runtime cpu usage of fastime daemon. Instead of getting the time from OS everytime, it calculates the time from last retrieval time and add the refresh interval of the daemon.

### CODE REVIEW

**Comment by kevindiu** [comment]
It reduce runtime cpu usage of fastime daemon. Instead of getting the time from OS everytime, it calculates the time from last retrieval time and add the refresh interval of the daemon.

**Comment by kevindiu** [comment]
New:
BenchmarkFastime-4   	2000000000	         1.01 ns/op	       0 B/op	       0 allocs/op
BenchmarkFastime-4   	2000000000	         1.01 ns/op	       0 B/op	       0 allocs/op
BenchmarkFastime-4   	2000000000	         0.99 ns/op	       0 B/op	       0 allocs/op
BenchmarkFastime-4   	2000000000	         0.99 ns/op	       0 B/op	       0 allocs/op
BenchmarkFastime-4   	2000000000	         1.00 ns/op	       0 B/op	       0 allocs/op

Original: 
BenchmarkFastime-4   	2000000000	         1.02 ns/op	       0 B/op	       0 allocs/op
BenchmarkFastime-4   	2000000000	         0.99 ns/op	       0 B/op	       0 allocs/op
BenchmarkFastime-4   	2000000000	         0.98 ns/op	       0 B/op	       0 allocs/op
BenchmarkFastime-4   	2000000000	         0.98 ns/op	       0 B/op	       0 allocs/op
BenchmarkFastime-4   	2000000000	         1.00 ns/op	       0 B/op	       0 allocs/op


Performance did not decreased a lot, but I think the runtime cpu usage will be decreased.

File: fastime.go [hunk 1]
```diff
@@ -48,17 +47,27 @@ func New() *Fastime {
...
+	f.update(time.Now())
+}
+
+func (f *Fastime) refreshTime() {
+	/*
+		lastNow := f.Now()
+		f.update(lastNow.Add(f.refInt))
+	*/
+	f.initTime()
 }
```
Comment (kpango) [inline fastime.go]: Why the initiTime and the refreshTime is same?

**Comment by kpango** [comment]
closed by https://github.com/kpango/fastime/pull/17

**Pull request closed without merging**